.github/workflows/PR-4.x.yaml

```diff
@@ -21,5 +22,9 @@ jobs:
   macOS-X64:
     uses: opencv/ci-gha-workflow/.github/workflows/OCV-PR-4.x-macOS-x86_64.yaml@main
 
+  iOS:
+    if: contains(github.event.pull_request.labels.*.name, 'ios')
```

Comment:
Seems `platform: ios/osx` label cannot be found using this condition. But if to use just `ios`, it works.

Reply:
Fixed it.